pyqtgraph/debug.py

```diff
@@ -126,7 +126,30 @@ def printTrace(msg='', indent=4, prefix='|'):
 
 def backtrace(skip=0):
     return ''.join(traceback.format_stack()[:-(skip+1)])    
+
+
+def formatException(exctype, value, tb, skip=0):
```

Comment:
`printException()` below uses `skip=1` to avoid having that function be the last entry in the printed stack trace.